components/suggestion/suggestion_box.jsx

```diff
@@ -337,6 +348,10 @@ export default class SuggestionBox extends React.Component {
         let handled = false;
         for (const provider of this.props.providers) {
             handled = provider.handlePretextChanged(this.suggestionId, pretext) || handled;
+
+            if (handled) {
+                break;
+            }
```

Comment:
Just to clean things up. no need to continue the loop if a provider has suggestions for the text given

Reply:
I had originally intended that you could have multiple providers active at once, but that hasn't been the case at all, and I assume that won't work anyway, so breaking out early makes sense